locales/en-US/app.l20n

```diff
@@ -104,8 +103,10 @@
 <contribute "Contribute">
 <nowActive "Now Active">
 <userCount title:"Active Users of this Experiment">
-
 <tourOnboardingTitle "{{title}} enabled!">
 <tourStartButton "Take the Tour">
 <tourCancelButton "Skip">
 <tourDoneButton "Done">
+<userCountContainerIntro "There are">
+<userCountContainerOutro "people trying {{title}} right now!">
```

Comment:
Random L10N, but does intro+outro play nice w/ rtl languages? Or should this be a single string with another variable somewhere in the middle?